hydra-plutus/src/Hydra/Contract/Head.hs

```diff
@@ -409,14 +411,22 @@ checkContest ctx contestationDeadline parties closedSnapshotNumber sig contester
   mustNotChangeParameters =
     traceIfFalse "changed parameters" $
       parties' == parties
-        && contestationDeadline' == contestationDeadline
         && headId' == headId
```

Comment:
MUST: Add `contestationPeriod' == contestationPeriod` here.

We must prevent a contester to reduce the period for future contesters.

Reply:
Actually `mustPushDeadline` is already validating that.

Not always will be that case.